django/conf/__init__.py

```diff
@@ -109,6 +110,28 @@ def configure(self, default_settings=global_settings, **options):
             setattr(holder, name, value)
         self._wrapped = holder
 
+    @staticmethod
+    def _add_script_prefix(value):
+        """
+        Prefix SCRIPT_NAME WSGI variable to ``value`` under certain circumstances.
+
+        Useful for when the app is being served at a subpath and manually prefixing
+        that subpath to STATIC_URL and MEDIA_URL in settings is inconvenient
+        (i.e., a deployment concern).
+        """
+        try:
+            URLValidator()(value)
+            return value
+        except (ValidationError, AttributeError):
+            pass
+
+        # Do not apply prefix if *_URL is set to absolute path
+        if value.startswith('/'):
+            return value
+        from django.urls import get_script_prefix  # circular dependency if top-level import
+        prefix = get_script_prefix()
+        return os.path.join(prefix, value.lstrip('/'))
```

Comment:
One minor change here, please don't use `os.path.join` but `'/'.join(…)`. `os.path.join` uses `os.sep` which doesn't have to be a slash always.

Reply:
Done